avalon/tools/cbsceneinventory/app.py

```diff
@@ -101,8 +101,39 @@ def _on_update_to_latest(items):
         menu.addAction(expandall_action)
         menu.addAction(collapse_action)
 
+        custom_actions = self.get_custom_actions(containers=items)
+        if custom_actions:
+            menu.addSeparator()
+            submenu = QtWidgets.QMenu("Actions")
+            submenu.setStyleSheet(module.window.styleSheet())
+            for Action in custom_actions:
+
+                action = Action()
+                color = Action.color or DEFAULT_COLOR
+                icon = qta.icon("fa.%s" % Action.icon, color=color)
+                action_item = QtWidgets.QAction(icon, Action.label, submenu)
+                action_item.triggered.connect(partial(action.process, items))
+
+                submenu.addAction(action_item)
+
+            menu.addMenu(submenu)
+
         return menu
 
+    def get_custom_actions(self, containers):
+        """Get the registered Inventory Actions"""
+
+        def sorter(value):
+            """Sort based on order"""
+            Plugin = value
```

Comment:
This could also just be
```python
        def sorter(Plugin):
            """Sort based on order"""
            return Plugin.order
```

Or is that more confusing?

Reply:
That line seems unnecessary.